web/satellite/src/components/common/List.vue

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="item-component">
+        <component
+            v-for="item in dataSet"
+            class="item-component__item"
+            :is="itemComponent"
+            :itemData="item"
+            @click.native="onItemClick(item)"
+            v-bind:class="[item.isSelected ? 'selected' : '']"/>
+    </div>
+</template>
+
+<script lang="ts">
+    import { Component, Prop, Vue } from 'vue-property-decorator';
+
+    @Component({
+        components: {},
```

Comment:
why do we have this empty decorator here?

Reply:
Good point